app/ConfigRepository.php

```diff
@@ -106,6 +113,11 @@ public function get($key, $default = null): mixed
             return $default;
         }
 
+        // Check if we need to load the OS YAML file
+        if (preg_match('/^os\.(?<os>[^.]+)/', $key, $matches)) {
```

Comment:
Probably could be optimized a little with starts_with() and substr()

Reply:
Done